prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -289,11 +302,14 @@ func getReviewers(rc reviewersClient, ghc githubClient, log *logrus.Entry, autho
 		fileReviewers := rc.Reviewers(file.Filename).Difference(authorSet)
```

Comment:
And the break just above ensures we only add non-leaf reviewers when we're missing some...